.ember-cli

```diff
@@ -5,5 +5,6 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": false
+  "disableAnalytics": false,
+  "port": 4202
```

Comment:
This has been added to run the app using "ember serve" earlier "ember serve" used to start the server on 4200, not it will start on 4202 for browse app.